introduction.md

```diff
@@ -0,0 +1 @@
+Hi! My name is Bill.  I am learning GIT so that I can introduce to my co-workers.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.